cts/cli/regression.crm_mon.exp

```diff
@@ -0,0 +1,2780 @@
+=#=#=#= Begin test: Basic text output =#=#=#=
+Cluster Summary:
+  * Stack: corosync
+  * Current DC: cluster02 (version) - partition with quorum
+  * Last updated:
+  * Last change:
+  * 5 nodes configured
+  * 18 resource instances configured
+
+Node List:
+  * Online: [ cluster01 cluster02 ]
+
+Active Resources:
+  * Clone Set: ping-clone [ping]:
+    * Started: [ cluster01 cluster02 ]
+  * Fencing	(stonith:fence_xvm):	 Started cluster01
+  * dummy	(ocf::pacemaker:Dummy):	 Started cluster02
+=#=#=#= Current cib after: Basic text output =#=#=#=
```

Comment:
Maybe we can modify the relevant function to not print the cib for this series

Reply:
In fact, we already had a way to do this - by passing an extra 0 argument to test_assert.  The file is much shorter now.